### ISSUE TITLE
Strict optional handling. 

### ISSUE THREAD

**Issue by ethanLhciactive** [jkHCBB/validator#1]
Was wondering if you had implemented a strict option that detects if an optional attribute has been spelled incorrectly.

**Comment by jkHCBB** [comment]
Currently the tool is using the same exact schema that CMS is using, So if the schema would pick that up then the tool should as well. If you have an example, I can try to get something in place but no promise it will happen this week. I built this for the sole purpose of reporting back all of the errors that the cms validator would fail an MRF for, so I never changed what it looks for.

**Comment by jkHCBB** [comment]
So to answer your question, If say plan_name was spelled plane_nam it would still pass

**Comment by ethanLhciactive** [comment]
yes that's accurate, or as an example from my file, "billing_code_modifier" is spelled "billing_code_modifiers" which when used with the -strict tag on the cms tool, would be flagged as an error.
